hkmc2/shared/src/test/mlscript/wasm/Operators.mls

```
@@ -0,0 +1,517 @@
+:wasm
+:wat
+2 + 1
+//│ Wat:
+//│ (module
+//│   (type (func (result (ref null any))))
+//│   (func $entry (type 0) (result (ref null any))
+//│     (local $plus_lhs (ref null any))
+//│     (local $plus_rhs (ref null any))
+//│     (block (result (ref null i31))
+//│       (local.set 0
+//│         (ref.i31
+//│           (i32.const 2)))
+//│       (local.set 1
+//│         (ref.i31
+//│           (i32.const 1)))
+//│       (if (result (ref null i31))
+//│         (i32.and
+//│           (ref.test (ref null i31)
+//│             (local.get 0))
+//│           (ref.test (ref null i31)
+//│             (local.get 1)))
+//│         (then
+//│           (ref.i31
+//│             (i32.add
+//│               (i31.get_s
+//│                 (ref.cast (ref null i31)
+//│                   (local.get 0)))
+//│               (i31.get_s
+//│                 (ref.cast (ref null i31)
+//│                   (local.get 1))))))
+//│         (else
+//│           (unreachable)))))
+//│   (export "entry" (func $entry))
+//│   (elem declare func $entry))
+//│ Wasm result:
+//│ = 3
+
+//│ Error: hkmc2.ErrorReport: Import of symbol `/Users/bakhtiyartemirov/Desktop/FYP/mlscript/hkmc2/shared/src/test/mlscript-compile/Predef.mjs` not implemented yet
```

Add a blank line above `:wasm:` on L1 to suppress this error.

---

Added this change in the latest commit, closing the comment as resolved.